pkg/k8shandler/trustedcabundle.go

```diff
@@ -0,0 +1,65 @@
+package k8shandler
+
+import (
+	"fmt"
+	"hash/fnv"
+	"strconv"
+
+	"github.com/openshift/cluster-logging-operator/pkg/utils"
+	"github.com/sirupsen/logrus"
+	core "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+)
+
+/*
+ * Create or update Trusted CA Bundle ConfigMap
+ * By setting label "config.openshift.io/inject-trusted-cabundle: true", the cert is automatically filled/updated.
+ */
+func (clusterRequest *ClusterLoggingRequest) createOrUpdateTrustedCABundleConfigMap(configMapName string) error {
+	logrus.Debug("createOrUpdateTrustedCABundlenfigMap...")
+	labels := map[string]string{
+		injectTrustedCABundleLabel: "true",
+	}
+	configMap := NewConfigMapWithLabels(
+		configMapName,
+		clusterRequest.cluster.Namespace,
+		labels,
+		map[string]string{
+			trustedCABundleKey: "",
+		},
+	)
+
+	utils.AddOwnerRefToObject(configMap, utils.AsOwner(clusterRequest.cluster))
+
+	err := clusterRequest.Create(configMap)
```

Comment:
If the CM already exists but needs to be updated where does that happen?  I think you need something like this https://github.com/openshift/cluster-logging-operator/blob/master/pkg/k8shandler/fluentd.go#L178-L192 if we really want to createOrUpdate which seems likely

Reply:
Instead, can we add a `CreateOrUpdateConfigmap` function to the `configmap.go` file and call that? (similar to https://github.com/openshift/cluster-logging-operator/blob/master/pkg/k8shandler/secret.go#L31)